Tests/LibGfx/TestImageWriter.cpp

```diff
@@ -107,6 +110,19 @@ TEST_CASE(test_bmp)
     TRY_OR_FAIL((test_roundtrip<Gfx::BMPWriter, Gfx::BMPImageDecoderPlugin>(TRY_OR_FAIL(create_test_rgba_bitmap()))));
 }
 
+TEST_CASE(test_gif)
+{
+    // We only support grayscale and non-animated images yet
+    auto bitmap = TRY_OR_FAIL(create_test_rgb_bitmap());
+
+    // Convert bitmap to grayscale
+    for (auto& argb : *bitmap)
+        argb = Color::from_argb(argb).to_grayscale().value();
+
+    bool const is_gif = true;
+    TRY_OR_FAIL((test_roundtrip<Gfx::GIFWriter, Gfx::GIFImageDecoderPlugin>(bitmap, is_gif)));
```

Comment:
I'll double check but the `TRY` macro usually requires one more pair of parentheses when using functions with multiple template parameters. Otherwise the comma to separate inbetween will turn them in parameters of the macro.

Reply:
It is indeed required.